src/vpk/Velopack.Packaging/PackageBuilder.cs

```diff
@@ -177,6 +177,16 @@ await ctx.RunTask(
                 if (TargetOs != RuntimeOs.Linux && portableTask != null) await portableTask;
                 if (setupTask != null) await setupTask;
 
+                if (!Options.NoInst && Options.BuildMsi && TargetOs == RuntimeOs.Windows) {
```

Comment:
Can we combine this into the existing setup task since it only applies to Windows?

Reply:
Done